lib/scripts/collection_list.rb

```diff
@@ -1,37 +1,41 @@
-def collections_path(collection)
-  starting_collection_data = collection_title_id_data(collection)
-  if collection.collections.empty?
-    collection_paths = [[starting_collection_data]]
-  else
-    collection_paths = []
+def build_collection_store_data
```

Comment:
For easier review of these changes, I will be renaming this file to correspond to this method in an upcoming commit.